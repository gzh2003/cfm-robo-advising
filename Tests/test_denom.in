SHOP.TO
AC.TO
ATVI
TD.TO
MRS.V
EPW.CN
FL.V
TSLA
AMD
AMZN
SOFI
AAPL