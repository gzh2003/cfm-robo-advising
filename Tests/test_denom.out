ATVI
TSLA
AMD
AMZN
SOFI
AAPL